functions/src/Android Version/index.ts

```diff
@@ -12,43 +12,52 @@ main.use(bodyParser.json());
 
 export const getAndroidVersion = functions.https.onRequest(main);
 
-app.get('/supportedAndroidVersions', (request, response) => {
-    getSupportedAndroidVersions(response);
+app.post('/supportedAndroidVersions', (request, response) => {
```

Comment:
FYI you may want to get Helena's express removal merged into master and then rebase this code.